test/runtime-test.js

```diff
@@ -947,7 +947,7 @@ describe('BEMHTML compiler/Runtime', function() {
       });
     });
 
-    it('should throw error with one apply', function() {
+    xit('should throw error with one apply', function() {
```

Comment:
since it depend on `this.options.debug` there is should be two ways of tests: with throwing and without